### PR TITLE
create header

### PR DESCRIPTION
**Pull request by jeremyjorand**

### CODE REVIEW

**Review by ThomasDCW — APPROVED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import "./App.css";
+import Header from "@components/Header";
 
 function App() {
   return (
     <div className="App">
       <p>coucou</p>
```
Comment (LuigiMenez) [inline frontend/src/App.jsx:7]: Don't forget to remove the "Coucou"

**Review by LuigiMenez — APPROVED** [review]
Used styled-components instead of app.css

File: frontend/src/App.css [hunk 2]
```diff
@@ -1,56 +1,6 @@
...
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
-
-button {
-  font-size: calc(10px + 2vmin);
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```
Comment (LoicBrassart) [inline frontend/src/App.css:1]: No CSS file, plz use styled-components

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
No vanilla css file !

**Review by ThomasDCW — APPROVED** [review]
looks ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Review by ThomasDCW — APPROVED** [review]
i fixed conflicts

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**